content/en/blog/_posts/2023-12-13-Canary-Deployment-With-Kubernetes/index.md

```diff
@@ -0,0 +1,336 @@
+---
+layout: blog
+title: "A Comprehensive Guide to Canary Deployment with Kubernetes"
+date: 2024-09-14
+slug: canary-deployment-with-kubernetes
+evergreen: true
+author: >
+  Mohammed Affan (AccuKnox)
+---
+
+Welcome to a comprehensive guide to canary deployment with Kubernetes.
+In modern software development getting quick feedback form the users
+is crucial for delivering a high-quality product and implementing agile
+delivery. Developers try to get feedback from the users without ruining the
+user experience. That's why canary deployment come into the picture.
```

Comment:
Either we should mention Istio in this introduction, or (better!) we should find a way to explain canary rollouts without depending on Istio.